lua/freeze/init.lua

```diff
@@ -82,6 +83,29 @@ M.parse_options = function(opts)
   return options
 end
 
+-- Open the generated image based on the OS
+---@param args FreezeOptions
+---@return string
+local function open_by_os(args)
+  local is_win = vim.loop.os_uname().sysname:match("Windows")
+  local output = vim.fn.expand(args.output)
+  local cmd = {}
+  if is_win then
+    table.insert(cmd, "explorer")
+  else
+    -- Maybe we should use xdg-open here too?
+    table.insert(cmd, "open")
+  end
```

Comment:
I'm happy to approve if we can get linux working too

Reply:
I don't have a working linux machine right now, only my Mac.

I know there are several options for opening files/images:

- `xdg-open` (GNOME)
- `gwenview` (KDE)
- `lximage-qt` (LXQT)
- And some non pre-installed like `feh`

But I don't know if `open`, if someone with Linux distro(s) can check it would be nice.

The implementation of this would result in something like this:

```lua
local open_commands = { "explorer", "open", "xdg-open", "gwenview", "lximage-qt", ... }
local open_cmd = ""
for _, command in ipairs(open_commands) do
  if vim.fn.executable(command) ~= 0 do
    open_cmd = command
  end
end
table.insert(cmd, open_cmd)
```

Reply:
I would opt for `xdg-open` since most machines have gnome installed or have `xdg-open` installed separately. `xdg-open` would also be better then using some other application like `feh` since if your mime settings are done correctly it should launch feh for you.